IBAnimatable/GradientType.swift

```diff
@@ -84,7 +87,7 @@ public enum GradientType: String, IBEnum {
   case redMist
   case tealLove
   case neonLife
-  case manOfSteel
+  case manofSteel
```

Comment:
Maybe we can improve it then, uppercasing next character after a space? It just feel weird otherwise 😆 
It would be interested to have the others though, @JakeLin @lastMove any opinions?